setup.py

```diff
@@ -2,7 +2,7 @@
 
 setuptools.setup(
       name='dpu_utils',
-      version='0.1.16',
+      version='0.1.17',
       license='MIT',

```

Comment:
I am really confused why this diff appears because I thought this was already merged